app/views/shared/check_answers/_client_details.html.erb

```diff
@@ -75,6 +75,28 @@
     <% end %>
   <% end %>
 
+   <% if applicant.applied_previously %>
+    <%= summary_list.with_row(html_attributes: { id: "app-check-your-answers__applied_previously" }) do |row| %>
+      <%= row.with_key(text: t(".previous_application_reference"), classes: "govuk-!-width-one-half") %>
+      <%= row.with_value(text: applicant.previous_reference) %>
+      <%= row.with_action(
+            text: t("generic.change"),
+            href: providers_legal_aid_application_previous_references_path,
```

Comment:
Think this should be providers_legal_aid_application_previous_references_path(@legal_aid_application) and that should fix some of the failing tests 🙂